tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,146 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
+
+Signals are easy.
+
+Make Signal (in sender):
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    signal SIGNALNAME
+
+ .. code-tab:: csharp
+
+     [Signal]
+     public delegate void SignalNameEventHandler();
```

Comment:
This should be technically called "Declare signal", but I don't really like the format here. Like @AThousandShips said:

> While concise this section is far too bare, this won't help anyone unless they know some basics, it needs some slightly less minimal descriptions, and it just repeats the same as below in simplified form, which I'd say is just redundant